### ISSUE TITLE
Build error: No module named 'utils.rst2dep'

### ISSUE THREAD

**Issue by lgessler** [amir-zeldes/gum#59]
When building GUM6 from a fresh clone of master, `process_reddit.py` works fine but I encounter the following error on running `_build/build_gum.py`:

```
» python _build/build_gum.py
====================
Validating files...
====================

Found reddit source data
Including reddit data in build
o Found 148 documents
o File names match
o Token counts match across directories
o 148 documents pass XSD validation                              
                                        Traceback (most recent call last):
  File "_build/build_gum.py", line 95, in <module>
    from utils.repair_rst import fix_rst
  File "/[...]/gum/_build/utils/repair_rst.py", line 6, in <module>
    from .rst2dep import make_rsd
ModuleNotFoundError: No module named 'utils.rst2dep'
```

**Comment by amir-zeldes** [comment]
I see it does exist in dev:

https://github.com/amir-zeldes/gum/blob/dev/_build/utils/rst2dep.py

I need to clean up and merge to master, thanks for spotting it.

**Comment by amir-zeldes** [comment]
Fixed in V6.1.0
